core/deployment/src/main/java/io/quarkus/deployment/builditem/FeatureBuildItem.java

```diff
@@ -11,6 +11,7 @@
 
     public static final String AGROAL = "agroal";
     public static final String KUBERNETES = "kubernetes";
+    public static final String KUBERNETES_CLIENT = "kubernetes-client";
```

Comment:
Well, while we are here, could we move them to respect the alphabetical order? AP4K got renamed, that's why it's here.

Reply:
Makes sense, fixed!